content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -30,7 +30,7 @@ The following arguments can be specified:
 
 - The `sort` argument is available for all databases on the `@unique`, `@@unique` and `@@index` attributes in version `3.5.0` and later. Additionally, SQL Server also allows it on `@id` and `@@id`.
 
-- The `type` argument is also available **in PostgreSQL only** on the `@@index` attribute in version `3.6.0` and later. This argument allows Prisma to support the `Hash` index access method as well as the default `BTree` access method.
+- The `type` argument is also available **in PostgreSQL only** on the `@@index` attribute in version `3.6.0` and later. This argument allows Prisma to support the `Hash` index access method as well as the default `BTree` access method. Version `3.14.0` adds additional access methods `Gist`, `Gin`, `SpGist` and `Brin`.
```

Comment:
```suggestion
- The `type` argument is also available **in PostgreSQL only** on the `@@index` attribute in version `3.6.0` and later. This argument allows Prisma to support the `Hash` index access method as well as the default `BTree` access method. 
- Prisma version `3.14.0` adds additional access methods with the index types `Gist`, `Gin`, `SpGist` and `Brin`.
```

Reply:
Each item in this list is one different argument to `@@index`. This item you broke up is the one for `type` (vs `sort` and `length` before for example). Instead of breaking this into another bullet we might want to rewrite the sentence. (Note that with the preview feature going away soon this will need to be rewritten anyway though and the version information can probably be moved somewhere else)

Reply:
Ah, yes, then let's not break it up! Let's leave it how it was but remove the version info. I'll do that as a suggestion.